### DISCUSSION TITLE
Conflict issue with CNAME records.

### DISCUSSION THREAD

**Discussion by AperatPeru** [community/community#43539] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello. I currently have a custom domain registered as www.example.org.pe. I have a repository on GitHub Pages that uses that domain to be published. On the other hand, I have a Google WorkSpace service in which I want to use personalized emails. Before I used GitHub to publish my page, the MX records and their subsequent verification were working fine, but then I realized that you weren't like that anymore and I couldn't receive my emails. After writing to Google and my domain provider, and even trying the Zoho Mail service, I've been told that the problem is the Github CNAME records they put up for my repository to redirect to my custom domain. I would like to know how to fix this problem and what is causing it. It's funny since I don't think my case is the first considering how popular the GitHub Pages service and the Google suite are (and any dedicated mail service like Zoho Mail).

**Comment by Kyaw-Zin-Latt** [comment]
![shwedagon](https://user-images.githubusercontent.com/59337230/211130540-56b1854e-d2a2-4b0b-8b24-b8471afca24e.jpg)

**Comment by jfullstackdev** [comment]
is this really your domain that you bought ? `http://www.example.org.pe/`

I doubt it's a personal domain

**Reply by Rickydenni** [reply]
Looking for my money from bitcoin trading platform

**Reply by AperatPeru** [reply]
No, it's just an example, but it should still work for asking for help on the problem itself.

**Comment by yoannchaudet** [comment]
I don't see anything wrong with www.aperat.org.pe (if that's the domain you are referring to).

I am also not aware of a `CNAME` record potentially impacting `MX` records and email delivery. If you can share verbatim what Google told you, we may be able to help further.

**Reply by AperatPeru** [reply]
Hello. Thanks for your reply. The technical support of my domain provider told me that it was the CNAME record that caused the problem, which is true because before I set up my page on GitHub Pages and used the Github CNAMEs, the emails were delivered and received correctly. But just yesterday I calmly read the support message again and noticed that it was only one CNAME record that was causing the problem, not both (and it seemed the less important of both), so I removed that CNAME without the fear of that my page fell, and immediately the emails working correctly again. In any case, it is clear that the CNAME record produced an error when receiving emails from my WorkSpace account, which was the one that omitted the "www." in the name of the record (the other did include the "www"). Keep that info because it's important.
However, I would like to know if such an error can be solved, since now if I write the name of my page in the address bar and omit the "www", the browser does not find it.

**Reply by yoannchaudet** [reply]
Ok I get it.

So `aperat.org.pe` (no `www` in front) is what's called an APEX domain (~ a root one). Typically `CNAME` are not allowed on such a domain. This is maybe what caused issues with your DNS provider. If you care about getting your site to route even when no `www` prefix is set, you can just use an `A` record for the APEX instead, like we document here: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages#using-an-apex-domain-for-your-github-pages-site.

**Reply by AperatPeru** [reply]
I am interested in that answer given. How should I populate the A records on my dashboard? Do I put the IP address? Because placing aperatperu.github.io does not accept it

**Reply by yoannchaudet** [reply]
Here is the step by step process: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain. Step 5 is what you are after especially. We recommend you create a `A` record for each of our 4 static IP addresses. You can additionally setup `AAAA` records if you want IPv6 support but it is optional.
